toonz/sources/toonz/sceneviewer.cpp

```diff
@@ -1019,8 +1019,10 @@ void SceneViewer::drawBackground()
 		PROC proc = wglGetProcAddress("glCheckFramebufferStatusEXT");
 		if (proc != nullptr)
 			status = reinterpret_cast<PFNGLCHECKFRAMEBUFFERSTATUSEXTPROC>(proc)(GL_FRAMEBUFFER);
-#else
+#elif MACOSX
 		status = glCheckFramebufferStatus(GL_FRAMEBUFFER);
+#else
+		status = 0;  /* XXX, stub */
```

Comment:
What’s the issue with querying the status of the fbo here, and in the next chunk?